dev-python/python-gnupg/python-gnupg-0.4.3.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6,3_7} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Python wrapper for GNU Privacy Guard"
+HOMEPAGE="https://pythonhosted.org/python-gnupg/ https://github.com/vsajip/python-gnupg/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+
+RDEPEND="app-crypt/gnupg"
+DEPEND="${RDEPEND}"
+
+# Tests are still hanging.
+RESTRICT="test"
+
+python_test() {
+	# Note; 1 test fails under pypy only
```

Comment:
Is this still the case? If so, please remove pypy from PYTHON_COMPAT.

Reply:
The don't hang for me, but instead, fail: https://pastebin.com/F3nKVbt9.
Sorry, I am not a python expert, so I don't have a clou..

Reply:
The fail with pypy for me, so removed.
-> https://pastebin.com/8Par3Ke0